File: base64_and_file/src/main/java/com/phone/base64_and_file/bean/Base64AndFileBean.java
package com.phone.base64_and_file.bean;

import android.graphics.Bitmap;

import java.io.File;
import java.util.ArrayList;
import java.util.List;

public class Base64AndFileBean {

    private String dirsPath;
    private String dirsPathCompressed;
    private String dirsPathCompressedRecover;
    private File file;
    private File fileCompressed;
    private File fileCompressedRecover;
    private Bitmap bitmap;
    private Bitmap bitmapCompressed;
    private Bitmap bitmapCompressedRecover;
    public String base64Str;
    private String txtFilePath;
    private List<String> base64StrList = new ArrayList<>();

    public String getDirsPath() {
        return dirsPath;
    }

    public void setDirsPath(String dirsPath) {
        this.dirsPath = dirsPath;
    }

    public String getDirsPathCompressed() {
        return dirsPathCompressed;
    }

    public void setDirsPathCompressed(String dirsPathCompressed) {
        this.dirsPathCompressed = dirsPathCompressed;
    }

    public String getDirsPathCompressedRecover() {
        return dirsPathCompressedRecover;
    }

    public void setDirsPathCompressedRecover(String dirsPathCompressedRecover) {
        this.dirsPathCompressedRecover = dirsPathCompressedRecover;
    }

    public File getFile() {
        return file;
    }

    public void setFile(File file) {
        this.file = file;
    }

    public File getFileCompressed() {
        return fileCompressed;
    }

    public void setFileCompressed(File fileCompressed) {
        this.fileCompressed = fileCompressed;
    }

    public File getFileCompressedRecover() {
        return fileCompressedRecover;
    }

    public void setFileCompressedRecover(File fileCompressedRecover) {
        this.fileCompressedRecover = fileCompressedRecover;
    }

    public Bitmap getBitmap() {
        return bitmap;
    }

    public void setBitmap(Bitmap bitmap) {
        this.bitmap = bitmap;
    }

    public Bitmap getBitmapCompressed() {
        return bitmapCompressed;
    }

    public void setBitmapCompressed(Bitmap bitmapCompressed) {
        this.bitmapCompressed = bitmapCompressed;
    }

    public Bitmap getBitmapCompressedRecover() {
        return bitmapCompressedRecover;
    }

    public void setBitmapCompressedRecover(Bitmap bitmapCompressedRecover) {
        this.bitmapCompressedRecover = bitmapCompressedRecover;
    }

    public String getBase64Str() {
        return base64Str;
    }

    public void setBase64Str(String base64Str) {
        this.base64Str = base64Str;
    }

    public String getTxtFilePath() {
        return txtFilePath;
    }

    public void setTxtFilePath(String txtFilePath) {
        this.txtFilePath = txtFilePath;
    }

    public List<String> getBase64StrList() {
        return base64StrList;
    }

    public void setBase64StrList(List<String> base64StrList) {
        this.base64StrList.clear();
        this.base64StrList.addAll(base64StrList);
    }
}

File: library_base64_and_file/src/main/java/com/phone/library_base64_and_file/manager/Base64AndFileBean.kt
package com.phone.library_base64_and_file.manager

import android.graphics.Bitmap
import java.io.File

class Base64AndFileBean {

    private var dirsPath: String? = null
    private var dirsPathCompressed: String? = null
    private var dirsPathCompressedRecover: String? = null
    private var file: File? = null
    private var fileCompressed: File? = null
    private var fileCompressedRecover: File? = null
    private var bitmap: Bitmap? = null
    private var bitmapCompressed: Bitmap? = null
    private var bitmapCompressedRecover: Bitmap? = null
    private var base64Str: String? = null
    private var txtFilePath: String? = null
    private val base64StrList: MutableList<String> = ArrayList()

    fun getDirsPath(): String? {
        return dirsPath
    }

    fun setDirsPath(dirsPath: String?) {
        this.dirsPath = dirsPath
    }

    fun getDirsPathCompressed(): String? {
        return dirsPathCompressed
    }

    fun setDirsPathCompressed(dirsPathCompressed: String?) {
        this.dirsPathCompressed = dirsPathCompressed
    }

    fun getDirsPathCompressedRecover(): String? {
        return dirsPathCompressedRecover
    }

    fun setDirsPathCompressedRecover(dirsPathCompressedRecover: String?) {
        this.dirsPathCompressedRecover = dirsPathCompressedRecover
    }

    fun getFile(): File? {
        return file
    }

    fun setFile(file: File?) {
        this.file = file
    }

    fun getFileCompressed(): File? {
        return fileCompressed
    }

    fun setFileCompressed(fileCompressed: File?) {
        this.fileCompressed = fileCompressed
    }

    fun getFileCompressedRecover(): File? {
        return fileCompressedRecover
    }

    fun setFileCompressedRecover(fileCompressedRecover: File?) {
        this.fileCompressedRecover = fileCompressedRecover
    }

    fun getBitmap(): Bitmap? {
        return bitmap
    }

    fun setBitmap(bitmap: Bitmap?) {
        this.bitmap = bitmap
    }

    fun getBitmapCompressed(): Bitmap? {
        return bitmapCompressed
    }

    fun setBitmapCompressed(bitmapCompressed: Bitmap?) {
        this.bitmapCompressed = bitmapCompressed
    }

    fun getBitmapCompressedRecover(): Bitmap? {
        return bitmapCompressedRecover
    }

    fun setBitmapCompressedRecover(bitmapCompressedRecover: Bitmap?) {
        this.bitmapCompressedRecover = bitmapCompressedRecover
    }

    fun getBase64Str(): String? {
        return base64Str
    }

    fun setBase64Str(base64Str: String?) {
        this.base64Str = base64Str
    }

    fun getTxtFilePath(): String? {
        return txtFilePath
    }

    fun setTxtFilePath(txtFilePath: String?) {
        this.txtFilePath = txtFilePath
    }

    fun getBase64StrList(): List<String> {
        return base64StrList
    }

    fun setBase64StrList(base64StrList: List<String>?) {
        this.base64StrList.clear()
        this.base64StrList.addAll(base64StrList!!)
    }
}